### ISSUE TITLE
CMF recordings from 1967 showing error

### ISSUE THREAD

**Issue by chesleyperry** [UCSCLibrary/dams_project_mgmt#340]
### Descriptive Summary

Links for 6 concerts (parent level records) and their children are showing an error in the DAMS:

All concerts are from 1967 -

Concert 2: https://digitalcollections.library.ucsc.edu/concern/works/6395w935g?locale=en
Concert 4: https://digitalcollections.library.ucsc.edu/concern/works/1n79h658j?locale=en
Concert 5: https://digitalcollections.library.ucsc.edu/concern/works/vd66w2265?locale=en
Concert 6: https://digitalcollections.library.ucsc.edu/concern/works/mk61rk29t?locale=en
Concert 7: https://digitalcollections.library.ucsc.edu/concern/works/12579v681?locale=en
Concert 8: https://digitalcollections.library.ucsc.edu/concern/works/2f75rb35b?locale=en

Child level links are also showing the same error:
Examples:
Last Four Songs from Richard Strauss: https://digitalcollections.library.ucsc.edu/concern/works/8049g747p?locale=en

Error message:

The page you were looking for doesn't exist.
You may have mistyped the address or the page may have moved.

### Acceptance Criteria

This is what done looks like

* [ ] All recordings from the 6 concerts are accessible

**Comment by chesleyperry** [comment]
Another detail that might help get to the bottom on this - Luisa was doing QA on July 17 and she said the first two recordings in the list were accessible then.

**Comment by NedHenry** [comment]
Thanks for noticing this Sue! I figured out what was going on. When I merged the old CMF collection from the initial test ingest with the new CMF collection from the full ingest, these works remained members of both collections (we just set the old collection to private so it didn't display to users). I recently deleted the old collection just to make the admin page clearer. I think these works are now trying to find a collection that isn't there. 
I should have it fixed soon.
